### ISSUE TITLE
missing space before year

### ISSUE THREAD

**Issue by VladimirAlexiev** [validatingrdf/validatingrdf.github.io#10]
Eg http://book.validatingrdf.com/bookHtml013.html#sec217:
```
Labra Gayo, Prud’hommeaux, Solbrig, and Boneva,2017 [58]
Hartmann, Zapilko, Wackerow, and Eckert,2016 [45]
```
There should be space before the year, eg `, 2017`

**Comment by labra** [comment]
Yes. Indeed, one problem there is that the original LaTeX code was using the [\bibentry](https://gking.harvard.edu/files/bibentry2.pdf) command which I think is not understood by Hevea. So I did the trick of redefining that command to show the authors and the year of a publication using citations. It works more or less, but I would prefer to show the title of the citation, and I don't know how to do that yet.
